db/data/regions/_default/en.yml

```diff
@@ -20,69 +21,77 @@ en:
             optional: Last name (optional)
           errors:
-            blank: Enter a last name
+            blank_instructional: Enter a last name
+            blank_informative: Last name is missing
             too_long: Last name is too long (maximum is 255 characters)
             contains_emojis: Last name cannot contain emojis
             contains_mathematical_symbols: Last name cannot contain mathematical symbols
             contains_restricted_characters: Last name can only contain letters, numbers,
               local characters, and special characters
-            contains_html_tags: Last name cannot contain HTML tags.
-            contains_url: Last name cannot contain a URL.
+            contains_html_tags: Last name can't contain HTML
+            contains_url: Last name can't contain a URL
         address1:
           label:
             default: Address
             optional: Address (optional)
           errors:
-            blank: Enter an address
+            blank_instructional: Enter an address
+            blank_informative: Address is missing
             too_long: The first address line is too long (maximum is 255 characters)
             contains_emojis: Address1 cannot contain emojis
             contains_mathematical_symbols: Address1 cannot contain mathematical symbols
             contains_restricted_characters: Address1 can only contain letters, numbers,
               local characters, and special characters
-            contains_too_many_words: Address1 cannot have more than %{word_count}
+            contains_too_many_words: Address1 must have fewer than %{word_count} words
               words
-            contains_html_tags: Address1 cannot contain HTML tags.
-            contains_url: Address1 cannot contain URLs.
-            missing_building_number: Add a building number if you have one.
-            street_unknown_for_zip: Enter a valid street name for %{zip}.
-            building_number_invalid: Building number couldn't be located for %{street},
-              %{zip}.
-            unknown_for_address: Address line 1 may be incorrect.
+            contains_html_tags: Address1 can't contain HTML
+            contains_url: Address1 can't contain a URL
+            missing_building_number_instructional: Add a building number if you have one
+            missing_building_number_informative: Building number may be missing
+            street_unknown_for_zip_instructional: Enter a valid street name for %{zip}.
+            street_unknown_for_zip_informative: Street name doesn't match %{zip}
+            building_number_invalid_instructional: Building number couldn't be located for %{street}, %{zip}
+            building_number_invalid_informative: Building number couldn't be located for %{street}, %{zip}
+            unknown_for_address: Address line 1 may be incorrect
           warnings:
             contains_too_many_words: Address line 1 is recommended to have less than %{word_count} words
```

Comment:
Are these also in reference to a field labelled simply as `Address`? Let's make sure the error is referencing the actual field label.